### PR TITLE
Fixed checking the width of the label in the navigator

### PR DESCRIPTION
**Pull request by SasLord**
<sub><a href="https://front.hc.engineering/guest/platform?token=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJsaW5rSWQiOiI2NjYwOWMwYmI3NTM3YzAxYTgxNTQ5NTYiLCJndWVzdCI6InRydWUiLCJlbWFpbCI6IiNndWVzdEBoYy5lbmdpbmVlcmluZyIsIndvcmtzcGFjZSI6InBsYXRmb3JtIiwicHJvZHVjdElkIjoiIn0.-OXCtcE-WySRiXHrAuHTEHElbVEhAJnEkABqfVhnk08">Huly&reg;: <b>UBERF-7167</b></a></sub>

### CODE REVIEW

File: packages/ui/src/resize.ts [hunk 1]
```diff
@@ -153,7 +153,7 @@ export const panelSeparators: DefSeparators = [
 ]
 
 export const workbenchSeparators: DefSeparators = [
-  { minSize: 12.5, size: 17.5, maxSize: 22.5, float: 'navigator' },
+  { minSize: 12.5, size: 17.5, maxSize: 35, float: 'navigator' },
```
Comment (haiodo) [inline packages/ui/src/resize.ts]: 40?

**Review by haiodo — DISMISSED** [review]

**Review by BykhovDenis — APPROVED** [review]

**Comment by kristina-fefelova** [comment]
Shouldn't we fix `bind:clientWidth`/`bind:clientHeight` in all places? (maybe in another pr)

**Comment by haiodo** [comment]
We have resizeObserver and usually not use them at all.

**Comment by kristina-fefelova** [comment]
I just see a couple of places with `bind:clientWidth`/`bind:clientHeight` and maybe we also need refactor them (use `resizeObserver` for example)

**Pull request merged**